l10n/en/firefox/channel.ftl

```diff
@@ -45,6 +45,7 @@ firefox-channel-download-and-test-the-latest-desktop = Download and test the lat
 firefox-channel-beta = { -brand-name-beta }
 firefox-channel-test-about-to-be-released = Test about-to-be-released features in the most stable pre-release build.
 firefox-channel-release-notes = Release Notes
+firefox-channel-previous-nightly-builds = Previous Nightly Builds
```

Comment:
Sorry I didn't reply sooner, it's been a crazy week.

Our brand names get special treatment for translation, and those names are kept in a separate file that we reference as variables in strings. We should format this string like:

```suggestion
firefox-channel-previous-nightly-builds = Previous { -brand-name-nightly } Builds
```